scripts/EmissionMATICXPolygonConfigurationDeploy.s.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IPermissionedPayloadsController} from 'aave-address-book/governance-v3/IPermissionedPayloadsController.sol';
+import {PermissionedControllerEmissionTestMATICXPolygon} from '../tests/PermissionedControllerEmissionTestMATICXPolygon.t.sol';
+import {Script} from 'forge-std/Script.sol';
+
+/**
+ * @dev Deploy Polygon
+ * deploy-command: make deploy-ledger contract=scripts/EmissionMATICXPolygonConfigurationDeploy.s.sol chain=polygon
+ */
+
+/**
+ * @dev Deploy Polygon
+ * deploy-command:
+ * make deploy-ledger contract=scripts/EmissionMATICXPolygonConfigurationDeploy.s.sol chain=polygon
+ * or
+ * make deploy-private-key contract=scripts/EmissionMATICXPolygonConfigurationDeploy.s.sol chain=polygon private_key=$\{PRIVATE_KEY\}
+ */
+contract EmissionMATICXPolygonConfigurationDeploy is
```

Comment:
This file is generated in the tests folder no? 

Also there are some example tests using the older flow in `tests` directory, I would replace them with some examples from new payloadsController flow (we can do that once we have the contracts deployed too)